sources/osg/Scissor.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+var MACROUTILS = require( 'osg/Utils' );
+var StateAttribute = require( 'osg/StateAttribute' );
+
+var Scissor = function ( x, y, w, h ) {
+    StateAttribute.call( this );
+
+    this._x = x !== undefined ? x : -1;
+    this._y = y !== undefined ? y : -1;
+    this._width = w !== undefined ? w : -1;
+    this._height = h !== undefined ? h : -1;
+};
+
+Scissor.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( StateAttribute.prototype, {
+
+    attributeType: 'Scissor',
+
+    cloneType: function () {
+        return new Scissor();
+    },
+
+    apply: function ( state ) {
+        var gl = state.getGraphicContext();
+        if ( this._x === -1 ) {
```

Comment:
yes true indeed.
(PR still "needs fixing", have to real test it before making it ready, (mutliple cam/viewport) obviously)